examples/collision/square-platform.cpp

```diff
@@ -289,30 +289,29 @@ void start() {
 
     int block_size = 50;
     colliders.push_back(&collider_screen);
-    SDL_Rect block1 = {.x=0,
-        .y=SCREEN_HEIGHT-block_size,
-        .w=2*block_size,
-        .h=block_size};
-    SDL_Rect block2 = {.x=SCREEN_WIDTH/4,
-        .y=SCREEN_HEIGHT-3*block_size,
-        .w=2*block_size,
-        .h=block_size};
-    SDL_Rect block3 = {.x=SCREEN_WIDTH/2,
-        .y=SCREEN_HEIGHT-4*block_size,
-        .w=2*block_size,
-        .h=block_size};
-    SDL_Rect block4 = {.x=SCREEN_WIDTH/6,
-        .y=SCREEN_HEIGHT-6*block_size,
-        .w=2*block_size,
-        .h=block_size};
-    SDL_Rect block5 = {.x=SCREEN_WIDTH/2,
-        .y=SCREEN_HEIGHT-8*block_size,
-        .w=2*block_size,
-        .h=block_size};
-    SDL_Rect block6 = {.x=SCREEN_WIDTH-2*block_size,
-        .y=SCREEN_HEIGHT-10*block_size,
-        .w=2*block_size,
-        .h=block_size};
+    SDL_Rect block1 = {
+        .x = 0, .y = SCREEN_HEIGHT - block_size, .w = 2 * block_size, .h = block_size
+    };
+    SDL_Rect block2 = { .x = SCREEN_WIDTH / 4,
```

Comment:
I think we can improve this as well, probably putting a comma in the end of the last struct field declaration.